contracts/oracle/CollateralizationMemoizer.sol

```diff
@@ -0,0 +1,204 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "./IOracle.sol";
+import "./ICollateralizationOracle.sol";
+import "../refs/CoreRef.sol";
+
+interface IPausable {
+    function paused() external view returns (bool);
+}
+
+/// @title Fei Protocol's Collateralization Oracle
+/// @author eswak
+/// @notice Reads a list of PCVDeposit that report their amount of collateral
+///         and the amount of protocol-owned FEI they manage, to deduce the
+///         protocol-wide collateralization ratio.
+contract CollateralizationMemoizer is IOracle, CoreRef {
```

Comment:
Can you rename CollateralizationOracleWrapper to be consistent with other naming?